apps/explorer/src/ui/Radio.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { RadioGroup as HeadlessRadioGroup } from '@headlessui/react';
+import { type ReactNode } from 'react';
+
+import { type ExtractProps } from '~/ui/types';
+
+export type RadioGroupProps = ExtractProps<typeof HeadlessRadioGroup> & {
+    children: ReactNode;
+    ariaLabel: string;
+};
+
+export function RadioGroup({ ariaLabel, children, ...props }: RadioGroupProps) {
+    return (
+        <HeadlessRadioGroup {...props}>
+            <HeadlessRadioGroup.Label className="sr-only">
+                {ariaLabel}
+            </HeadlessRadioGroup.Label>
+            {children}
+        </HeadlessRadioGroup>
+    );
+}
+
+export type RadioOptionProps = ExtractProps<
+    typeof HeadlessRadioGroup.Option
+> & {
+    label?: string;
+    children?: ReactNode;
+};
+
+export function RadioOption({ label, children, ...props }: RadioOptionProps) {
+    return (
+        <HeadlessRadioGroup.Option
+            className="item-center ui-check:text-hero-dark flex cursor-pointer flex-col rounded-md border border-transparent bg-white text-steel-dark hover:text-steel-darker ui-checked:border-steel"
```

Comment:
@mamos-mysten It is set to `text-hero-dark` here already.

Reply:
hmm... it doesn't seem to be working in storybook: 

<img width="747" alt="image" src="https://user-images.githubusercontent.com/122397493/231302532-e2f213eb-3a32-4513-a4cf-0d78c55782b5.png">

also noticed we have an ARIA violation at the bottom - looks like we might need a `role=` attribute somewhere

Reply:
Ah ok I see the problem here. The `Label` needs its own styling. @mamos-mysten

Reply:
one more thing I noticed - looks like you have `item-center` instead of `items-center` here

Reply:
Removed, pls take a look again @mamos-mysten